### PR TITLE
Add aldjemy recipe

### PR DESCRIPTION
**Pull request by ltalirz**
recipe for the [aldjemy](https://pypi.python.org/pypi/aldjemy/0.6.0) package.

This recipe is for aldjemy 0.6.0, which we need for our materials science package.
I can add another recipe for the latest version 0.7.0 later on.

### CODE REVIEW

File: recipes/aldjemy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sqlalchemy >0.7.1
```
Comment (synapticarbors) [inline recipes/aldjemy/meta.yaml]: I don't think `sqlalchemy` is a build requirement of this package. Please remove it.

File: recipes/aldjemy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - django <1.11
+    - django-extensions
+    - sqlalchemy >0.7.1
+
+  source_files:
+    - test_project
+
+about:
+  home: https://github.com/Deepwalker/aldjemy/
+  license: BSD
```
Comment (synapticarbors) [inline recipes/aldjemy/meta.yaml]: Could you raise an issue on the source repo asking them to add a license file and include it in the `MANIFEST.in` and then copy a link to that issue here?

File: recipes/aldjemy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    - sqlalchemy >0.7.1
+
+test:
+  imports:
+    - aldjemy
+
+  commands:
+    # django 1.11 yields
+    # ImportError: No module named test1.urls 
+    - 'cd test_project; python manage.py test'  # [not win]
```
Comment (synapticarbors) [inline recipes/aldjemy/meta.yaml]: Could you add an equivalent windows test?

**Comment by synapticarbors** [comment]
In terms of updating to `0.7.0` later, once this staged recipe is approved and a feedstock is created, please put in a PR on that feedstock to update to the latest version.

**Comment by ltalirz** [comment]
Following @synapticarbors suggestion, an [issue](https://github.com/Deepwalker/aldjemy/issues/49) has been raised and the package maintainer is happy to add the license file.
See also this [pull request](https://github.com/Deepwalker/aldjemy/pull/50).

**Pull request merged**